doc/embedded/crosscompile_example.md

```diff
@@ -0,0 +1,194 @@
+## Crosscompile mlpack example for an embedded hardware
+
+In this article, we explore how to crosscompile and run an mlpack example code
+on an embedded hardware. In our previous
+[documentations](crosscompile_armv7.cmake), we have explored how to run mlpack
+bindings such as kNN command line program on a Raspberry PI 2. Please refer to that
+article first and follow the first part on how to setup cross-compilation toolchain,
+and then continue with this article. Or refer to this
+[table](supported_boards.md) for a quick start guide.
```

Comment:
So just to be clear, it seems like this example is focused on using CMake to set up a project that will do embedded cross-compilation?

Reply:
yes indeed, this is the main idea, because you need a guide if you have your own code